composer.json

```diff
@@ -19,17 +19,17 @@
         "php": ">=7.0.8",
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "drupal-composer/drupal-scaffold": "^2.5",
+        "drupal/core-composer-scaffold": "8.8.x-dev",
+        "drupal/core-recommended": "^8.7.0",
         "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7.0",
```

Comment:
The change above made `zaporylie/composer-drupal-optimizations` fail to determine which version of Symfony components can be removed while calculating dependencies. So either you add config manually as suggested in https://github.com/zaporylie/composer-drupal-optimizations#configuration or remove that dependency completely as it no longer makes sense to have it here.

Not sure what `drupal/core-recommended` is - new core subsplit? - but I guess I can modify `zaporylie/composer-drupal-optimizations` to also read version constraints from it. https://github.com/zaporylie/composer-drupal-optimizations/blob/master/src/Plugin.php#L23

Reply:
Thanks for noticing and commenting, @zaporylie.

Yes, what we are talking about here is a new set of subtree splits that are maintained by the drupal.org infrastructure and replace the existing community-maintained metapackages (e.g. webflo/drupal-core-strict and webflo/drupal-core-dev-dependencies).

A diagram showing the relationship between the different new metapackages can be found in the [drupal.org infrastructure package generator README](https://bitbucket.org/drupalorg-infrastructure/package-generator/src/master/README.md). It would be great if you could support projects that use these metapackages in zaporylie/composer-drupal-optimizations.

Reply:
If I read this correctly `drupal/core-recommended` is just `drupal/core` with exact version constraints for all core's required packages. Since versioning convention of the new package strictly follows `drupal/core` versions, the only thing that needs to be done is reading `drupal/core-requirements` version constraint if dependency present in project's composer.json.

https://github.com/zaporylie/composer-drupal-optimizations/pull/15/files
I'm planning to merge it later today or on Friday if no one has any objections.

Reply:
@zaporylie 👍 thanks for the release!